cypress/e2e/tests/migration/applicationinventory/manageimports/csv.test.ts

```diff
@@ -16,16 +16,18 @@ limitations under the License.
 /// <reference types="cypress" />
 
 import { login, openManageImportsPage } from "../../../../../utils/utils";
-import { topKebabMenu, kebabMenuItem } from "../../../../views/applicationinventory.view";
+import { kebabMenuItem } from "../../../../views/applicationinventory.view";
+import { Application } from "../../../../models/migration/applicationinventory/application";
 
 describe(["@tier2"], "Manage imports tests", function () {
     before("Login", function () {
         login();
+        Application.open();
         openManageImportsPage();
     });
 
     it("Download CSV template", function () {
-        cy.get(topKebabMenu).eq(1).click();
+        cy.get('button[aria-label="Actions"]').eq(1).click();
```

Comment:
can 'button[aria-label="Actions"]'  be stored as a variable in the view file ?

Reply:
Solved